libcodechecker/server/api/report_server.py

```diff
@@ -1921,19 +1921,18 @@ def massStoreRun(self, name, tag, version, b64zip, force):
                 LOG.debug("Storing source file: " + source_file_name)
 
                 if not os.path.isfile(source_file_name):
-                    # The file was not in the ZIP file, because we already have the
-                    # content. Let's check if we already have a file record in the
-                    # database or we need to add one.
+                    # The file was not in the ZIP file, because we already
```

Comment:
Changes in this file irrelevant to the PR, but the commit failed to generate locally for me on pep8 errors:

```
libcodechecker/server/api/report_server.py:1924:80: E501 line too long (83 > 79 characters)
libcodechecker/server/api/report_server.py:1925:80: E501 line too long (82 > 79 characters)
libcodechecker/server/api/report_server.py:1934:80: E501 line too long (80 > 79 characters)
```

Reply:
I fixed them in #1166 too.

Reply:
Okay, undid the changes.